src/model.js

```diff
@@ -0,0 +1,242 @@
+const yelp = require("yelp-fusion");
+const proj4 = require("proj4");
+const _ = require("lodash");
+const async = require("async");
+
+const {
+  koopProviderYelp: { api_key },
+} = require("config");
+const client = yelp.client(api_key);
+const YELP_API_MAX_RADIUS = 40000;
+
+const proj = proj4("GOOGLE", "WGS84");
+
+function Model() {}
+
+// each model should have a getData() function to fetch the geo data
+// and format it into a geojson
+Model.prototype.getData = function (req, callback) {
+  if (req.query && req.query.returnCountOnly) {
+    return callback(null, {
+      type: "FeatureCollection",
+      count: 2001,
```

Comment:
Why is there a hard-coded number?

Reply:
This is [how the provider worked before](https://github.com/koopjs/koop-provider-yelp/blob/bb26d2735b95183441fbe7087004205268504ae5/yelp.js#L13-L15). I had assumed that was there for a specific purpose, but maybe it's not?

Reply:
I'd remove it for now then.

Reply:
👍